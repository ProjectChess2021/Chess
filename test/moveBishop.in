game human human
move d2 d3
move g7 g6
move c1 h6
move f8 h6