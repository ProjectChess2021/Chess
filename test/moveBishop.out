8 rnbqkbnr      MoveHistory
7 pppppppp
6  _ _ _ _
5 _ _ _ _
4  _ _ _ _
3 _ _ _ _
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 pppppppp      White: pawn d2 -> d3
6  _ _ _ _
5 _ _ _ _
4  _ _ _ _
3 _ _P_ _
2 PPP PPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 pppppp_p      White: pawn d2 -> d3
6  _ _ _p_      Black: pawn g7 -> g6
5 _ _ _ _
4  _ _ _ _
3 _ _P_ _
2 PPP PPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 pppppp_p      White: pawn d2 -> d3
6  _ _ _pB      Black: pawn g7 -> g6
5 _ _ _ _       White: bishop c1 -> h6
4  _ _ _ _
3 _ _P_ _
2 PPP PPPP
1 RN_QKBNR

  abcdefgh
8 rnbqk_nr      MoveHistory
7 pppppp_p      White: pawn d2 -> d3
6  _ _ _pb      Black: pawn g7 -> g6
5 _ _ _ _       White: bishop c1 -> h6
4  _ _ _ _      Black: bishop f8 -> h6 (capture a bishop)
3 _ _P_ _
2 PPP PPPP
1 RN_QKBNR

  abcdefgh
